2-mandatory/1-create-functions.js

```diff
@@ -44,7 +50,13 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arrayOfNumbers) {
+  return arrayOfNumbers.map((number) => {
+    if(number > 100){
```

Comment:
I would simplify this line of code to:
```
if(number > 100) {
    number = 100;
}
```
or
```
number = number > 100 ? 100 : number;
```
In this way, the adding of percentage is done on line 58 only